admin/react/assets/src/dashboard-app/pages/welcome/ListItem.js

```diff
@@ -0,0 +1,233 @@
+			return <link rel="stylesheet" id={`bcf-custom-font-${fontItem.id}-css`} href={`https://fonts.googleapis.com/css?family=${fontName}&ver=${fontItem.id}`} media="all"></link>
+		}
+	}
+
+	const getRenderFontWeight = (weight) => {
+		return weight.replace( "italic", "" );
+	}
+
+	const getFontAssetLink = (type, font, weight, version) => {
+		if ( type === 'local' ) {
+			return '';
+		}
+		const fontName = font.replace( / /g, "+" );
+		return <link rel='stylesheet' id={`bcf-google-font-${version}-link`} href={`https://fonts.googleapis.com/css?family=${fontName}:${weight}&display=fallback&ver=${version+1}`} media='all' />
+	}
+
+	const getFontStyle = (type, style, weight) => {
+		let considerStyler = ( '' === type || 'local' === type ) ? style : weight;
+		if ( '' === considerStyler ) {
+			return 'normal';
+		}
+		if ( considerStyler.includes('italic') ) {
+			return 'italic';
+		}
+		return style;
+	}
+
+	return (
+		<>
+			<div className={`${active || checkDelete ? "bg-white" : ""} transition-colors`}>
+				<div className="flex items-center justify-between py-5 border-b border-light list-font-title">
+					{ getAssetDetail(item) }
+					<div className="flex items-center px-6 mobile:block">
+						<h1 className="text-xl" style={{  fontFamily: item.title, fontWeight: "normal", fontSize: "2em" }}> {item.title} </h1>
+						<div className="sm:ml-3 mobile:mt-3 text-sm"> {`(${item['fonts-data']['variations'] ? item['fonts-data']['variations'].length : 0} ${__( 'variants', 'custom-fonts' )})`} </div>
+					</div>
+					<div className="flex px-6">
+						{checkDelete ? (
+							<div className="flex gap-x-6">
+								<div className="text-secondary cursor-pointer">
+									{__('Remove', 'custom-fonts') + ' "' + item.title + '" ' + __('font?', 'custom-fonts')}
+								</div>
+								<div
+									onClick={() => setCheckDelete(false)}
+									className="text-neutral cursor-pointer"
+								>
+									{__('Cancel', 'custom-fonts')}
+								</div>
+
+								<div
+									className="text-danger cursor-pointer"
+									onClick={deleteFontPost}
+									data-font_id={item.id}
+									data-font_type={item['font-type']}
+									data_font_name={item.title}
+								>
+									{__('Remove', 'custom-fonts')}
+								</div>
+							</div>
+						) : (
+							<div className="flex gap-x-6">
+								<div
+									onClick={setupEditFontScreen}
+									data-font_id={item.id}
+									data-font_type={item['font-type']}
+									data_font_name={item.title}
+									className="text-primary cursor-pointer"
+								>
+									{__('Edit', 'custom-fonts')}
+								</div>
+
+								<div
+									onClick={() => setCheckDelete(true)}
+									className="text-danger cursor-pointer"
+								>
+									{__('Remove', 'custom-fonts')}
+								</div>
+							</div>
+						)}
+
+						<div
+							onClick={() => setActive(!active)}
+							className="sm:ml-11 mobile:ml-2 cursor-pointer"
+						>
+							<svg
```

Comment:
Same here @imnavanath regarding svg reuse

Reply:
Added separate task in backlog for this - https://brainstormforce.atlassian.net/browse/CFP-38